packages/cli/src/utils/config.ts

```diff
@@ -23,10 +24,19 @@ export function getConfig(resave = true): I18nConfig | null {
   return result;
 }
 
-export function saveConfig(config: I18nConfig) {
+export async function saveConfig(config: I18nConfig) {
   const configFilePath = _getConfigFilePath();
 
-  const serialized = JSON.stringify(config, null, 2);
+  const prettierConfigFilePath = await prettier.resolveConfigFile(configFilePath) || ''
+
```

Comment:
this is wrong, configFilePath is the location of the i18n.json file, please see line #47.

Reply:
also, prettier must find its config file itself: it can be in multiple formats, also it can be specified as a field in package.json, so let's let it do its job

Reply:
Thank you for pointing that out! I see now that specifying configFilePath isn’t necessary since it points to i18n.json, and Prettier can locate its config file starting from process.cwd() on its own. I’ll remove this and let Prettier handle the config detection, as you suggested.